src/zope/configuration/xmlconfig.py

```diff
@@ -100,10 +117,40 @@ def __str__(self):
             return str(v)
 
 class ParserInfo(object):
-    """Information about a directive based on parser data
+    """
+    Information about a directive based on parser data
 
     This includes the directive location, as well as text data
     contained in the directive.
+
+    Example
+
+        >>> from zope.configuration.xmlconfig import ParserInfo
+        >>> info = ParserInfo('tests//sample.zcml', 1, 0)
```

Comment:
The doubled slashes confuse me.

(I'm sorry, I don't know how to review patches that *move code around*, all I see is newly-added code!)